### ISSUE TITLE
Disconnect Issue ES v3.2.3

### ISSUE THREAD

**Issue by n8suX** [rpuig2001/InitialCimbPlugin#2]
First of all, i love your plugin!

Since version 3.2.3 InitialClimbPlugin causes disconnects of AudioForVatsim. Maybe you can update your plugin? 

Greets!

**Comment by rpuig2001** [comment]
Hi! Thank you.
When does it crashes?
Do you have any confirmation that the issue is fue this plugin.

Roger.

**Comment by n8suX** [comment]
When i connect with ES, randomly between 2 and 10 Minutes, all aircrafts disappear, and appear again. then AFV disconnects. Im still online with ES and can control, but on VatSpy, vPilot/ xPilot, i will be shown as offline. 

i unload your plugin, since ES is working again with no issues.

sorry for my bad english!:)

**Comment by rpuig2001** [comment]
Thanks,
I'll check it as soon as I get some time

**Comment by rpuig2001** [comment]
Hi @n8suX,
Can you confirm that the issue is still there?

**Comment by n8suX** [comment]
@rpuig2001 yes, Same issue

**Comment by rpuig2001** [comment]
That's something weird. Nothing was changed and we didn't have any related problem so far.

**Comment by n8suX** [comment]
Did you Test this plugin with the current release?

**Comment by rpuig2001** [comment]
Of course, no more releases since NOV 2021 have been released. For that reason nothing has changed...

**Comment by rpuig2001** [comment]
Which Euroscope and/or beta version are you using?

**Comment by n8suX** [comment]
ES latest beta 3.2.3
